src/dialogs/more-info/ha-more-info-logbook.ts

```diff
@@ -40,7 +40,12 @@ export class MoreInfoLogbook extends LitElement {
     }
     const stateObj = this.hass.states[this.entityId];
 
-    if (!stateObj) {
+    if (!stateObj || stateObj.attributes.unit_of_measurement) {
+      return html``;
+    }
+
+    const domain = computeStateDomain(stateObj);
+    if (domain === "sensor" || domain === "proximity") {
```

Comment:
Can we move these to a const in `data/logbook.ts`?

Reply:
yes, done